infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/ILogReplicationConfigAdapter.java

```diff
@@ -30,4 +31,6 @@ public interface ILogReplicationConfigAdapter {
      * to the replicated data, therefore, this data must be provided by the plugin externally.
      */
     Map<UUID, List<UUID>> getStreamingConfigOnSink();
+
+    void setCorfuStore(CorfuStore corfuStore);
```

Comment:
We may need to find an alternative way of doing this as introducing this method will make the current implementation from outside fail